source/_docs/http-to-https.md

```diff
@@ -60,9 +60,15 @@ Use [Terminus](/docs/terminus) to run `wp search-replace` to converts URLs from
 ```
 terminus remote:wp <site>.<env> -- search-replace 'http://www.example.com' 'https://www.example.com' --all-tables --verbose
 ```
+##### Via Dashboard
+Another option that will work for most WordPress sites is to clone the Live database down to Test - and then clone it back to Live, ensuring you select the desired URL conversion and to Convert URLs' Protocol to HTTPS.
+
 #### Drupal
 Drupal 7 sites can use [Drush Search and Replace (sar)](https://www.drupal.org/project/sar) by [adding custom Drush command](/docs/drush/#add-custom-drush-commands). Drupal 8 sites can use the [Entity API](https://www.drupal.org/docs/8/api/entity-api/introduction-to-entity-api-in-drupal-8) to fetch data from entities in the database that may include insecure references.
 
+#### Via Dashboard
+Another option that will work for most sites is to clone the Live database down to Test - and then clone it back to Live, ensuring you select the desired 
```

Comment:
@lauren-kelly looks like this bit wasn't quite finished. would this be accurate? 
> Another option that will work for most sites is to clone the Live database down to Test - and then clone it back to Live, ensuring you select the desired values for URL conversion.

Reply:
Ah, on second glance. Maybe this section was an accident? Seems duplicate to the first on lines 63-65